contracts/helpers/MockCarbonPOL.sol

```diff
@@ -0,0 +1,9 @@
+// SPDX-License-Identifier: SEE LICENSE IN LICENSE
+pragma solidity 0.8.13;
+
+/**
+ * @dev mock carbon POL contract to receive pool surplus tokens
+ */
+contract MockCarbonPOL {
```

Comment:
shouldn't it be able to accept ETH as well?

Reply:
yup, added a receive method